packages/forma-36-react-components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -21,6 +24,7 @@ export const Checkbox = ({
       type={type}
       willBlurOnEsc={willBlurOnEsc}
       {...otherProps}
+      ref={undefined}
```

Comment:
Is this necessary?

Reply:
It was before but after I changed the ControlledInputProps to Omit<BoxProps<'div'>, 'ref'>`. It works :) Note that `ref` is still automatically typed by using `forwardRef`.